packages/earn-controller/src/EarnController.ts

```diff
@@ -422,12 +561,13 @@ export class EarnController extends BaseController<
     }
 
     const { eligible: isEligible } =
-      await this.#stakingApiService.getPooledStakingEligibility([
+      await this.#earnApiService.pooledStaking.getPooledStakingEligibility([
         addressToCheck,
       ]);
 
     this.update((state) => {
       state.pooled_staking.isEligible = isEligible;
+      state.lending.isEligible = isEligible;
```

Comment:
Perhaps we ought to document in the changelog that `refreshStakingEligibility` now stores `isEligible` under `state.lending` as well as `state.pooled_staking`?